draft/README.md

```diff
@@ -33,6 +33,9 @@ intended to be used for broadcasting the UAV flight data in real time. It can
 either be used by UAV, by add-on modules attached to a UAV, by the UAV ground
 station or by ground receivers and relays.
 
+Although radio specifications are described, this protocol may be used to 
```

Comment:
I wonder if this regards the protocol itself.

I guess your point is that you want to be able to send the payloads in that exact format when using an Internet connection to send your positions in the French law context, right? If I'm right, so the answer is that this will be definitively possible. :)

Or shall we split the actual protocol in two: one for the payload binary format itself, and one for the radio/transmission?